src/styled/Homepage.styled.js

```diff
@@ -73,19 +86,25 @@ export const StyledRecommendedList = styled.div`
   height: max-content;
 
   overflow-x: auto;
-  /* hide scrollbar but allow scrolling */
-  /* & {
-		-ms-overflow-style: none; 
-		scrollbar-width: none; 
-		overflow-x: scroll;
-	}
-
-	&::-webkit-scrollbar {
-		display: none; 
-	} */
-`;
 
-// gonna reuse this somehow ****************
+  @media screen and (min-width: 1024px) {
+    ::-webkit-scrollbar {
+      height: 10px;
+    }
+    ::-webkit-scrollbar-track {
+      background: rgba(245, 182, 66, 0.2);
+      border-radius: 100vw;
+      margin-inline: 8px;
+    }
+    ::-webkit-scrollbar-thumb {
+      background: var(--dt-radial-gradient);
+
+      border-radius: 100vw;
+    }
+    ::-webkit-scrollbar-thumb:hover {
+    }
+  }
+`;
```

Comment:
Can we move this to index.css or app.css to have every scrollbar styled in the app?

Reply:
Firefox just doesnt support that fancy styling,its webkit browsers only.You can only change the color of thumb and track in firefox.There is a fallback in index.css for firefox, does it work? should just be orange thumb and pale orange track